urls.tsv

```diff
@@ -1,5 +1,5 @@
 # Id	Version	Platform	Architecture	Upstream Url	Extension	sha256sum	Use upstream
-AI-FANN	1.10	src	all	http://www.cpan.org/authors/id/S/SA/SALVA/AI-FANN-0.10.tar.gz	.tar.gz	6d4f46cdde08ab001a13132db8bdda0e419ddce1d8bed10bd75bdaf84211f28f	True
+AI-FANN	1.1	src	all	http://www.cpan.org/authors/id/S/SA/SALVA/AI-FANN-0.10.tar.gz	.tar.gz	6d4f46cdde08ab001a13132db8bdda0e419ddce1d8bed10bd75bdaf84211f28f	True
```

Comment:
Looks like an unintentional change?

Sorry but could you please revert the removal of the `.0` since that will affect all of those packages.